exp-player/addon/components/exp-free-response/component.js

```diff
@@ -140,7 +140,14 @@ export default ExpFrameBaseComponent.extend(Validations, {
     },
     loadData: function(frameData) {
         var responses = frameData.responses;
-        this.set('EventTime', responses.EventTime);
+        var times = this.get('times');
+        var eventTime = null;
+        for (var i=0; i < times.length; i++) {
+            if (times[i]['24h'] === responses.EventTime) {
+                eventTime = times[i];
```

Comment:
Fine as is.

Purely for future reference, since you only expect there to be one match, you can save the computer a little work by using the `break` keyword inside the loop if a match if found.

> "The break statement terminates the current loop, switch, or label statement and transfers program control to the statement following the terminated statement."